docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -285,14 +285,13 @@ You can customize the scan by entering custom rules or other rulesets to scan wi
 
 To add Semgrep into your Buildkite pipeline:
 
-1. Create or edit a `pipeline.yml` configuration file to add a Semgrep command as part of your pipeline. Refer to the [Buildkite code snippet](#sample-buildkite-configuration-snippet). This configuration file can also be stored within Buildkite.
-2. Copy the relevant code snippet provided in [Sample Buildkite configuration snippet](#sample-buildkite-configuration-snippet).
-3. If you are using Buildkite to store the configuration, save the updated file. Otherwise, commit the updated configuration file into the `/.buildkite` folder within the target repository.
-4. The Semgrep job starts automatically upon detecting the committed `pipeline.yml` file. You can also view the job through Buildkite's interface, by clicking **your repository > Pipelines**.
-5. Optional: Create a separate CI job for diff-aware scanning, which scans only changed files in PRs or MRs, by repeating steps 1-3 and uncommenting the `SEMGREP_BASELINE_REF` definition provided within the code snippet.
+1. Prepare a configuration file to add a Semgrep scan as part of your pipeline. This configuration file can be stored within Buildkite or as a `pipeline.yml` file in the related repository.
+2. Copy the code snippet provided in [Sample Buildkite configuration snippet](#sample-buildkite-configuration-snippet), making alterations if necessary for your environment.
+3. If you are using Buildkite to store the configuration, save the updated file. Otherwise, commit the updated `pipeline.yml` file into the `/.buildkite` folder within the target repository.
+4. The Semgrep job starts automatically upon detecting the committed `pipeline.yml` file, or if you are using the Buildkite UI, you can select **New build**. You can view the job through Buildkite's interface by clicking **your repository > Pipelines**.
```

Comment:
```suggestion
4. The Semgrep job starts automatically upon detecting the committed `pipeline.yml` file. Alternatively, if you are using the Buildkite UI, you can select **New build**. You can view the job through Buildkite's interface by clicking **your repository > Pipelines**.
```

Reply:
Gonna add this, but I also realized the second instruction isn't quite right in the current BK UI, so I'll have to add another adjustment too.